<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RsaToolbox::getPortsDialog</class>
 <widget class="QDialog" name="RsaToolbox::getPortsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Ports...</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>284</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="portsList">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>256</width>
     <height>201</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>Port 1</string>
    </property>
    <property name="checkState">
     <enum>Checked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Port 2</string>
    </property>
    <property name="checkState">
     <enum>Checked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Port 3</string>
    </property>
    <property name="checkState">
     <enum>Checked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Port 4</string>
    </property>
    <property name="checkState">
     <enum>Checked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Port 5</string>
    </property>
    <property name="checkState">
     <enum>Checked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Port 6</string>
    </property>
    <property name="checkState">
     <enum>Checked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Port 7</string>
    </property>
    <property name="checkState">
     <enum>Checked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Port 8</string>
    </property>
    <property name="checkState">
     <enum>Checked</enum>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="selectAllCheckbox">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>28</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>selectAllCheckbox</tabstop>
  <tabstop>portsList</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>getPortsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>getPortsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
